internal/system/agent/router.go

```diff
@@ -63,7 +65,7 @@ func operationHandler(w http.ResponseWriter, r *http.Request) {
 	err = o.UnmarshalJSON(b)
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusBadRequest)
-		LoggingClient.Error("error during decoding: %v", err.Error())
+		LoggingClient.Error("error during decoding", "errMsg", err.Error())
```

Comment:
Avoid multiple keys (for example, here we have _two_: `error during decoding` and `errMsg`). Collapse them like so:
```
LoggingClient.Error("error during decoding: %v", err.Error())
```

Or simply:
```
LoggingClient.Error(err.Error())
```